packages/libs/components/src/plots/PlotlyPlot.tsx

```diff
@@ -95,6 +100,8 @@ function PlotlyPlot<T>(
     spacingOptions,
     showSpinner,
     showNoDataOverlay,
+    showExportButton,
+    exportFileName,
```

Comment:
Sanity check:
At first i thought it could be helpful to set a default `exportFileName` to cover us in the case of `showExportButton=true` but no `exportFileName` is given. However, since our `PlotlyPlot` component is only eventually called by `makePlotlyPlotComponent` which requires a `displayName` that becomes `exportFileName`, we should never hit the situation of an emty `exportFileName`. 
Yes?

Reply:
There is a default in the button component: https://github.com/VEuPathDB/web-monorepo/blob/203e823ebbcb920f36cbc17846aebd2d3f1a3c62/packages/libs/components/src/plots/ExportPlotToImageButton.tsx#L20